core/geometry/src/test/polyface/Polyface.test.ts

```diff
@@ -674,351 +583,367 @@ function flattenGeometry(...data: GeometryData[]): GeometryQuery[] {
-
-      y0 += 200.0;
-      allCountsA.push(countsA);
-      allCountsB.push(countsB);
-      allCountsC.push(countsC);
-    }
-    GeometryCoreTestIO.consoleLog({ angleCounts: allCountsA, chordCounts: allCountsB, maxEdgeLengthCounts: allCountsC });
-    GeometryCoreTestIO.saveGeometry(allGeometry, "Polyface", "SphereDensity");
-    expect(ck.getNumErrors()).equals(0);
-  });
-  it("Boxes", () => {
-    const allBox = flattenGeometry(Sample.createBoxes(false), Sample.createBoxes(true));
-    writeAllMeshes(allBox, "Box", true, allOptions, y0Box, optionYStep);
-  });
-  it("TorusPipes", () => {
-    const allBox = Sample.createTorusPipes();
-    writeAllMeshes(allBox, "TorusPipe", true, allOptions, y0TorusPipe, optionYStep);
-  });
-  it("LinearSweeps", () => {
-    // writeAllMeshes(Sample.createSimpleLinearSweeps(), "LinearSweepSubset", allEOptions, -y0LinearSweep, optionYStep);
-    writeAllMeshes(Sample.createSimpleLinearSweeps(), "LinearSweep", true, allOptions, y0LinearSweep, optionYStep);
-  });
+const options0 = new StrokeOptions();
+const optionsE = new StrokeOptions();
+const optionsN = new StrokeOptions();
+const optionsP = new StrokeOptions();
+const optionsPN = new StrokeOptions();
+const optionsPNE = new StrokeOptions();
+
+optionsE.maxEdgeLength = 0.5;
+optionsN.needNormals = true;
+optionsP.needParams = true;
+optionsPN.needNormals = true;
+optionsPN.needParams = true;
+optionsPNE.maxEdgeLength = 0.5;
+optionsPNE.needNormals = true;
+optionsPNE.needParams = true;
+const allOptions = [options0, optionsN, optionsP, optionsE, optionsPNE];
+
+const bigYStep = 800.0; // step between starts for different solid types
+const optionYStep = 100.0; // steps between starts for option variants of same solid type
+const y0OpenSweeps = 0.0;
+const y0ClosedSampler = y0OpenSweeps + bigYStep;
+const y0Box = y0ClosedSampler + bigYStep;
+const y0Cone = y0Box + bigYStep;
+const y0Sphere = y0Cone + bigYStep;
+const y0TorusPipe = y0Sphere + bigYStep;
+const y0LinearSweep = y0TorusPipe + bigYStep;
+const y0RotationalSweep = y0LinearSweep + bigYStep;
+const y0RuledSweep = y0RotationalSweep + bigYStep;
+
+it("Cones", () => {
+  const all = Sample.createCones();
+  writeAllMeshes(all, "Cone", true, allOptions, y0Cone, optionYStep);
+});
+it("Spheres", () => {
+  const all = Sample.createSpheres();
+  writeAllMeshes(all, "Sphere", true, allOptions, y0Sphere, optionYStep);
+});
+it("Boxes", () => {
+  const allBox = flattenGeometry(Sample.createBoxes(false), Sample.createBoxes(true));
+  writeAllMeshes(allBox, "Box", true, allOptions, y0Box, optionYStep);
+});
+it("TorusPipes", () => {
+  const allBox = Sample.createTorusPipes();
+  writeAllMeshes(allBox, "TorusPipe", true, allOptions, y0TorusPipe, optionYStep);
+});
+it("LinearSweeps", () => {
+  writeAllMeshes(Sample.createSimpleLinearSweeps(), "LinearSweep", true, allOptions, y0LinearSweep, optionYStep);
+});
 
```

Comment:
Again just indentation fix. I did not have time to go over the tests. Maybe in the future!